tree/dataframe/src/RLoopManager.cxx

```diff
@@ -556,13 +618,21 @@ void RLoopManager::CleanUpTask(unsigned int slot)
 /// This method also clears the contents of GetCodeToJit().
 void RLoopManager::Jit()
 {
+   // TODO this should be a read lock unless we find GetCodeToJit non-empty
    R__LOCKGUARD(gROOTMutex);
 
    const std::string code = std::move(GetCodeToJit());
-   if (code.empty())
+   if (code.empty()) {
+      R__LOG_INFO(RDFLogChannel()) << "Nothing to jit and execute.";
       return;
+   }
 
+   TStopwatch s;
+   s.Start();
```

Comment:
So you are going to measure the jitting time always, not just when you are logging? Does this happen once per event loop?

Reply:
yes, but `Start` and `Stop` are basically just taking a timestamp, they are cheap (esp. compared to typical jitting or event loop times)